### DISCUSSION TITLE
How to convert a loha safetensor trained from diffusers to webui format

### DISCUSSION THREAD

**Discussion by JIAOJIAYUASD** [huggingface/diffusers#7823] [Q&A]
Hello, when I finetune SDXL (actually that is InstantID) with PEFT method, I use lora、loha and lokr for  PEFT in [diffuser](https://github.com/huggingface/diffusers).
I have a question, how to convert a loha safetensor trained from diffusers to webui format？
In the training process:
the loading way:
`peft_config  = LoHaConfig(
            r=args.rank,
            alpha=args.rank //2,
            target_modules=["to_k", "to_q", "to_v", "to_out.0"],
        ) `
`unet = get_peft_model(unet, peft_config)
`
when train process finished, the saving way as:
`unet.save_pretrained(args.output_dir)`

and I get the safetensor as
![image](https://github.com/KohakuBlueleaf/LyCORIS/assets/61881733/f71caa9a-4935-40f8-84fb-0a18d19991ac)

But [webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui/) can't recognize it, I can't use it in webui.

How can I fix this promblem!

**Comment by asomoza** [comment]
I don't fully understand all that you wrote, but if you're training the LoRA with one of the diffusers scripts, you can use  the `--output_kohya_format` arg to save it.
